src/services/ActivitiesService.ts

```diff
@@ -18,7 +19,7 @@ class ActivitiesService {
    * Retrieves Activities based on the provided parameters
    * @param params - getActivities query parameters
    */
-  public getActivities(params: any): Promise<any> {
+  public getActivities(params: any): Promise<ActivitySchema[]> {
```

Comment:
Can this params any be turned into an object the same way it is in other services